### DISCUSSION TITLE
How to add disqus comment session in gatsby-theme-minimal-blog?

### DISCUSSION THREAD

**Discussion by tomvictor** [LekoArts/gatsby-themes#728] [Questions]
I have tried the official doc below, but looks like it doesn't work for me,

https://www.gatsbyjs.com/docs/how-to/adding-common-features/adding-comments/

Thanks

**Comment by LekoArts** [comment]
Please see https://github.com/LekoArts/gatsby-themes/discussions/698 - you'll want to shadow the PostFooter (https://github.com/LekoArts/gatsby-themes/blob/main/themes/gatsby-theme-minimal-blog/src/components/post-footer.tsx) component and there you can place the comment section.

**Comment by ch4nd4n** [comment]
Refer to this Git diff - https://github.com/ch4nd4n/chandankumar.com/commit/2cda31ab71b66d73391565d8a6da233057bacff8?diff=unified

Or you can refer to my blog post on this at - https://www.chandankumar.com/tutorial/gatsby-disqus-comments

https://github.com/ch4nd4n/chandankumar.com/blob/2cda31ab71b66d73391565d8a6da233057bacff8/src/%40lekoarts/gatsby-theme-minimal-blog/components/post.tsx

Note that I did not try @LekoArts suggesting on adding post-footer.tsx
